src/Umbraco.Web.UI.Client/src/assets/lang/en.ts

```diff
@@ -2189,9 +2191,11 @@ export default {
 		validateAsEmail: 'Validate as an email address',
 		validateAsNumber: 'Validate as a number',
 		validateAsUrl: 'Validate as a URL',
+		validationPattern: 'Validation pattern',
 		enterCustomValidation: '...or enter a custom validation',
 		fieldIsMandatory: 'Field is mandatory',
 		mandatoryMessage: 'Enter a custom validation error message (optional)',
+		mandatoryMessageLabel: 'Feedback message (Optional)',
```

Comment:
I think we should remove "(Optional)" from the label to match the same UX as the rest of the Backoffice. We mark mandatory fields with an `*` and do nothing special for the others.